invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -10,6 +11,21 @@ const get = documentPid => {
   });
 };
 
+const setRestrictionsOnDocument = (documentPid, accessList) => {
+  const operation = isEmpty(accessList) ? 'remove' : 'add';
```

Comment:
I am not sure here you have taken into account the case of "replacing", or do you?

What about this example:
* I have `nico@cern.ch`
* I remove it
* I add `nico2@cern.ch`
* I click on the save button.

Is `nico@cern.ch` replaced with `nico2@cern.ch`?

Reply:
Yes, it will be replaced; in essence, if it appears the current access list you see, it will be in the access